src/Microsoft.AspNetCore.Authentication.OpenIdConnect/Events/IOpenIdConnectEvents.cs

```diff
@@ -38,17 +28,22 @@ public interface IOpenIdConnectEvents : IRemoteAuthenticationEvents
         /// <summary>
         /// Invoked before redirecting to the identity provider to authenticate.
         /// </summary>
-        Task RedirectToAuthenticationEndpoint(RedirectContext context);
+        Task RedirectToIdentityProvider(RedirectContext context);
```

Comment:
Hmm, we do call it RedirectToAuthorizationEndpoint for all of our other providers.